proposals/PROPOSALS.md

```diff
@@ -1,141 +1,125 @@
+In other words, volunteers should be recruited who have knowledge of and experience with at least the following:
+ * Using Haskell in a variety of environments (research, traditional business applications, free-time open-source work, large teams, small teams)
+ * Using different build tools and dependency management techniques
+ * Domain knowledge such as cryptography, security, performance, and databases
+The ED will report on the current state of the committee's coverage of these areas as membership is updated.
+Applications to join the committee will be decided on by the committee's members.
+Inactive members may be removed by the ED after making a good-faith effort to contact them.
+
+The current members of the TWG are:
+ * Davean Scies (@davean)
+ * Gershom Bazerman (@gbaz)
+ * John Ericson (@Ericson2314)
+ * Hécate (@Kleidukos)
+ * Evie Ciobanu (@eviefp)
+ * David Thrane Christiansen (@david-christiansen)
+
+## Decisions
+
+The committee will make all decisions using a simple majority without secret ballots.
+Discussions and vote tallies regarding potential committee members are to be kept strictly confidential, while other discussions and votes should be transparent to the public.
+Decisions should be made with the participation of 2/3 of the current members of the committee, rounded up.
+The committee may decide to hold regular meetings, or to collaborate asynchronously, or some mix of the two.
+For instance, a valid decision can be made by having a non-quorum meeting discuss a matter, summarize the results for the other members, and then vote by email based on the summary.
+Disputes will be resolved by the ED.
+
+## Process
+
+The TWG, as an advisory committee to the Haskell Foundation and to the community at large, exists primarily to aid in making good decisions. 
+
+All proposals should address the following concerns:
+ * **Problem statement:** What problem is the proposed work intended to solve? What are the requirements against which a solution should be evaluated? How will solving this problem benefit the community?
+ * **Prior art:** What other similar work has been done in the community, and how is the proposal related to it?
+ * **Related efforts:** What other related activities are planned or ongoing? How is the proposal connected to them?
+ * **Technical and organizational work**: What work is actually being proposed? What technology should be developed, and how will it be managed and cared for going forward?
+ * **Success:** What does it mean for the project to have succeeded?
+ * **Stakeholders:** Who will be affected, and how?
+ * **Time:** How long will the proposal take to implement?
+
+Additionally, projects seeking HF support or execution should additionally address:
+ * **Budget:** How much time and money will the project cost initially? What about ongoing maintenance?
```

Comment:
Add
* **Who will do the work**

Reply:
Sounds good, though I'd rather have it for all proposals, including RFCs (as the proposal author may be part of a larger organization, and it's useful to clarify whether they personally will do the work or whether their organization will do so)

Reply:
This has been added under the "Technical and organization work" bullet point, as it seems natural to describe the actors immediately following the roles.